### ISSUE TITLE
Budgeting demo seems to be broken and reveals deeper library issues

### ISSUE THREAD

**Issue by GoldenChrysus** [silevis/reactgrid#410]
**Describe the bug**
The first demo I am introduced to as a potential user of this library is your budgeting demo, but it does not appear to be functioning correctly as the aggregations are inaccurate and the functionality does not align with docs' claims.

**Current behavior**
- The quarterly and yearly aggregations are incorrect.
  - 2020/Q1 displays as 453,990 despite the monthly values totaling 99,310.
  - This is just one example of several incorrect aggregations.
- Dragging the bottom corner of an aggregation cell into the cell below it appears to have some impact on the values/calculations as the numbers correct themselves when doing so.
  - This also occurs when opening an aggregation cell's context menu and choosing Cut.

**Expected behavior**
- The aggregations be correct from the beginning.
- The demo page states "Only white cells are able to change their value" (I assume "change their value" means "be edited by the user" as all cells may "change their value" -- especially aggregations) but dragging the corner of an aggregation is having some material impact on the values in the table. I would expect, based on this statement, to not be able to use modifying actions (like dragging a cell corner) in the first place, much less cause the values in the table to change by doing so. Even if the values are incorrect, they should not change when a user interacts with the cell (if the docs are to be believed).

Unfortunately with ReactGrid's own demo showing that these fundamental concepts are unreliable, I could never consider this project for production use. Are the demos at least not undergoing tests for validity? The fact that 20,380 + 23,480 + 55,450 does not equal 453,990, in a _financial_ example, should be a very obvious issue that a test should have revealed.

**Screenshots or gifs**
![](https://i.imgur.com/vjJ9RGg.gif)

**Your environment details**
 - Device: Desktop
 - OS: Windows 10
 - Browser: Vivaldi

**Comment by MichaelMatejko** [comment]
I agree. We are working on a new version of RractGrid and together with it we will publish a new website with new samples

**Comment by MichaelMatejko** [comment]
The issue is caused by code outside of the library. The lib itself doesn't have any baked in mechanism for calculations.
